client/nginx/default.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen       8080;
+    server_name  localhost;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+    }
+
+    location /search/ {
+        proxy_pass {{{ES_END_POINT}}};
```

Comment:
I think you need to allow only harmless HTTP methods here...

```sh
curl -XPUT https://search-new-eval.demo.cfpb.gov/search/im-in-ur-elasticsearch
{"acknowledged":true,"shards_acknowledged":true,"index":"im-in-ur-elasticsearch"}
```

Reply:
@rosskarchner It's a great point. I'm assuming this setup came from [here](https://github.com/cfpb/github-wiki-search/pull/113/files#diff-72f3e2d92b15895972bfa32294e38e33L51), but what I don't know is whether the old version of the app in production had a better or more-secure config. @OrlandoSoto might be able to track that down so we could find out if the old version of the app was proxying everything. I don't have access to the box.

Reply:
@rosskarchner We confirmed that this matches the existing config, unfortunately, so will leave it as-is for now. But we'll note it as a potential future improvement. Thanks for flagging this.

Reply:
👍